containers/unit_tests/TestDualView.hpp

```diff
@@ -622,6 +622,22 @@ TEST(TEST_CATEGORY,
             const_dv.view<Kokkos::DefaultExecutionSpace>());
 }
 
+TEST(TEST_CATEGORY, dualview_of_view) {
```

Comment:
Isn't this more about the use of the `SequentialHostInit` view alloc property?
The inner element type doesn't need to be views nor contain views, does it?

Reply:
No, it doesn't need to be a View of Views but it makes for a better test (and that one kind of needs `SequentialHostInit` so I rather called it `dualview_of_view` instead of `dualview_sequential_host_alloc` but I don't feel strongly about the name).